uint16 speed
---
roof status
